src/bun.js/bindings/JSBuffer.cpp

```diff
@@ -2333,6 +2337,158 @@ JSC_DEFINE_HOST_FUNCTION(jsBufferPrototypeFunction_hexSlice, (JSGlobalObject * l
     return jsBufferPrototypeFunction_SliceWithEncoding<WebCore::BufferEncodingType::hex>(lexicalGlobalObject, callFrame);
 }
 
+template<typename I> void write_int64_le(uint8_t* buffer, I value)
+{
+    static_assert(std::endian::native == std::endian::little);
+    auto val = reinterpret_cast<uint8_t*>(&value);
+    buffer[0] = val[0];
```

Comment:
```suggestion
   memcpy(value, &value, sizeof(value))
```

Reply:
this compiles to exactly the same code and thought it looked more clear to mirror the _be version

Reply:
![image](https://github.com/user-attachments/assets/e8a491c1-0017-4bbb-bc36-ff0d50943145)